app/src/js/components/DeleteCollection/DeleteCollectionModal.js

```diff
@@ -40,7 +37,7 @@ class DeleteCollectionModal extends React.Component {
         <p> You have submitted a request to delete the following collection: </p>
         <br />
         <h1>
-          <strong>{collectionLabelForId(this.props.collectionLabel)}</strong>
+          <strong>${this.props.collectionLabel}</strong>
```

Comment:
So I see this extra "$" in the prompt.

Reply:
Which only shows up on the single collection page when you go to delete. 
Not on the collections list page when you can select multiple to delete at a time.

Reply:
Oh, yes, that's bad.  https://github.com/nasa/cumulus-dashboard/pull/1051/commits/b2a1d5799dfed34f633332a98f70b568b5a06199